lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2668,6 +2670,7 @@ IsAccessorTransparentRequest::evaluate(Evaluator &evaluator,
 
   switch (accessor->getAccessorKind()) {
   case AccessorKind::Get:
+  case AccessorKind::DistributedGet:
```

Comment:
Distributed getters shouldn't be transparent.

Reply:
ok